posix.mak

```diff
@@ -107,6 +108,31 @@ else
 endif
 
 ################################################################################
+# Ddoc build variables
+################################################################################
+DDOC_VARS_STABLE=\
```

Comment:
If we agree on [renaming _STABLE to _LATEST](https://github.com/dlang/dlang.org/pull/1689), this should be changed as well.
However without #1689 it the naming is `stable`...